src/components/InlineCheckbox/InlineCheckbox.js

```diff
@@ -68,6 +72,7 @@ export default class InlineCheckbox extends React.Component {
       name,
       onClick,
       onKeyDown,
+      title = null,
```

Comment:
Wondering if we can keep the default `undefined` - Though I think `null` should work, too, just thought we'd avoid snapshot change :thinking:.